README.md

```diff
@@ -323,6 +323,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Barkatul Mujauddin](https://github.com/barkatul)
   - [Barphie](https://github.com/barphie)
   - [Benjamin Rukundo](https://github.com/rukundob451)
+  - [Bipul Sharma](https://github.com/itsmebipul00)
```

Comment:
Please add your name in line 338, after Bineet Naidu.
Let us know if you need any help doing that! 🙂


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable